### DISCUSSION TITLE
How to change Git Signs?

### DISCUSSION THREAD

**Discussion by Integralist** [nvim-neo-tree/neo-tree.nvim#1251] [Q&A]
👋🏻 How can I override this empty square (which is indicating a file has changed from what git is tracking)...

<img width="267" alt="Screenshot 2023-12-01 at 16 43 15" src="https://github.com/nvim-neo-tree/neo-tree.nvim/assets/180050/8a653c35-811c-4e74-89de-e2d743dfc33a">

Thanks

**Comment by wigging** [comment]
After reading the help section at `:h neo-tree-git-status`, I ended up with the configuration shown below. Notice the `git_status` table is where you can define the symbols. The actual symbols may not show up on here if you are not using a nerd font but this is how you configure the symbols.

```lua
return {
  "nvim-neo-tree/neo-tree.nvim",
  branch = "v3.x",
  dependencies = {
    "nvim-lua/plenary.nvim",
    "nvim-tree/nvim-web-devicons",
    "MunifTanjim/nui.nvim"
  },
  opts = {
    default_component_configs = {
      name = { use_git_status_colors = false },
      git_status = {
        symbols = {
          -- Change type
          added     = "",
          deleted   = "",
          modified  = "",
          renamed   = "",
          -- Status type
          untracked = "",
          ignored   = "",
          unstaged  = "",
          staged    = "",
          conflict  = "",
        }
      }
    },
    filesystem = {
      filtered_items = {
        hide_dotfiles = false,
        never_show = { ".DS_Store" }
      }
    }
  }
}
```
